src/CalcManager/CEngine/Rational.cpp

```diff
@@ -182,14 +182,20 @@ namespace CalcEngine
         return *this;
     }
 
+    /// <summary>
+    /// Calculate the remainder after division, the sign of the result will match the sign of the current object.
+    /// </summary>
+    /// <remarks>
+    /// This function has the same behavior than the standard C/C++ operator '%', to calculate the modulus after division instead, use <see cref="Rational::operator%"/> instead.
```

Comment:
```suggestion
    /// This function has the same behavior as the standard C/C++ operator '%.
    /// To calculate the modulus after division instead, use <see cref="Rational::operator%"/>.
```

Reply:
The remarks for both comments added in this file confuse me.  They both redirect to `operator%` but they are _describing_ `operator%`!  (circular reference).  Did you intend to direct me somewhere else?

Reply:
You are right! Reference fixed!